backend/blueprint/api/controllers/homePage.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, jsonify
+from utils.response import Response
+
+router = Blueprint('homePage', __name__)
+
+
+@router.route('/', methods=['POST'])
```

Comment:
Why this is POST?

Reply:
I was testing services while creating them using `cURL` therefore I kept it as `POST`

Reply:
yes I and @ParthS007  will be working on this portion while you refractor them.

Reply:
Alright then. :+1: